packages/fxa-settings/package.json

```diff
@@ -23,7 +23,7 @@
     "restart": "npm run build-postcss && pm2 restart pm2.config.js",
     "stop": "pm2 stop pm2.config.js",
     "build": "NODE_ENV=production npm run build-postcss && INLINE_RUNTIME_CHUNK=false rescripts build",
-    "test": "rescripts test",
+    "test": "SKIP_PREFLIGHT_CHECK=true rescripts test",
```

Comment:
Ahaha you did this too! Was it because of a babel-loader (or similar) mismatch?

Reply:
I had to add this because it wouldn't run tests without it and suggested adding this🤔

Reply:
Same!